$OpenBSD: patch-configure,v 1.1 2019/05/23 20:44:24 naddy Exp $

Honor the result of AC_SEARCH_LIBS() and don't hardcode linking with
libstdc++.

Index: configure
--- configure.orig
+++ configure
@@ -19858,7 +19858,7 @@ echo "$as_me: error: Required library id3lib not found
 fi
 
 
-    LIBS="-lz -lstdc++ $LIBS"
+#   LIBS="-lz -lstdc++ $LIBS"
 
 
 cat >>confdefs.h <<\_ACEOF
